go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/elastic/cloud-sdk-go
 
-go 1.12
+go 1.13
```

Comment:
I was sure we were already on 1.13! 😬

Reply:
Yeah, this is enforcing that now the minimum version required for this SDK is 1.13, but it could also be 1.14. Now 1.12 is not supported for this library